pkg/sdk/validation/manifest/json_remote_implementation.go

```diff
@@ -3,13 +3,20 @@ package manifest
 import (
```

Comment:
Please also remember about validating all our manifests against this validator and fix found issues 🙂 I know that there are some, because I tested this functionality against Hub manifests.

Ideally if these pull request(s) with fixes are created before second review, as they are prerequisites for this PR to be merged.